src/Pelagos/Entity/ResearchGroup.php

```diff
@@ -293,6 +293,15 @@ class ResearchGroup extends Entity
      */
     protected $datasets;
 
+    /**
+     * Lock flag for Research Group.
+     *
+     * @var boolean $lockStatus
+     *
+     * @ORM\Column(type="boolean", nullable=false)
+     */
+    protected $lockStatus = false;
```

Comment:
Status in lockStatus but be a bit ambiguous. Maybe we should just call it 'locked' ?